diff --git a/home/hyro/Workspace/transiq/node_modules/gtfs/.bun-tag-bd38f97cde78fba3 b/.bun-tag-bd38f97cde78fba3
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/dist/index.js b/dist/index.js
index 4043d1ba73277cc24e958930073337f4c1775c54..764dc43eed14f91fafe5e8d32711882948919946 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -3975,14 +3975,11 @@ var vehicles = {
 
 // src/lib/db.ts
 import fs from "fs";
-import Database from "better-sqlite3";
+import Database from "bun:sqlite";
 import untildify from "untildify";
 var dbs = {};
 function setupDb(sqlitePath) {
   const db = new Database(untildify(sqlitePath));
-  db.pragma("journal_mode = OFF");
-  db.pragma("synchronous = OFF");
-  db.pragma("temp_store = MEMORY");
   dbs[sqlitePath] = db;
   return db;
 }
@@ -4815,11 +4812,11 @@ var createGtfsTables = (db) => {
       if (column.type === "time") {
         sqlColumnCreateStatements.push(
           `${getTimestampColumnName(column.name)} INTEGER GENERATED ALWAYS AS (
-            CASE 
-              WHEN ${column.name} IS NULL OR ${column.name} = '' THEN NULL 
+            CASE
+              WHEN ${column.name} IS NULL OR ${column.name} = '' THEN NULL
               ELSE CAST(
-                substr(${column.name}, 1, instr(${column.name}, ':') - 1) * 3600 + 
-                substr(${column.name}, instr(${column.name}, ':') + 1, 2) * 60 + 
+                substr(${column.name}, 1, instr(${column.name}, ':') - 1) * 3600 +
+                substr(${column.name}, instr(${column.name}, ':') + 1, 2) * 60 +
                 substr(${column.name}, -2) AS INTEGER
               )
             END
